releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,21 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.0.0 - October 3rd, 2024
+### Breaking Changes
+* Increased the default maximum retry count when resolving Key Vault references and increased the minimum refresh interval to 1 minute. This helps prevent throttled requests to Key Vault and will reduce the chance of failing to resolve a secret due to momentary failures. [#589](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/589)
+
+### Enhancements
+* **Variant Feature Flags**: This is the first stable release with support for the `variants`, `allocation`, and `telemetry` properties for feature flags that were introduced in 7.1.0-preview. This enables [experimentation with Azure App Configuration](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-experimentation), which opens up possibilities for improving user experience or software functionality using data from your feature flags.
+* This is the first stable release of the load balancing mode introduced in 8.0.0-preview.3.
```

Comment:
Can we repeat the release note here so we don't force users to go back and look for details from the previous preview release?

Reply:
Sounds good